kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/config/TargetCluster.java

```diff
@@ -17,7 +18,7 @@
 /**
  * Represents the target (upstream) kafka cluster.
  */
-public record TargetCluster(@JsonProperty(value = "bootstrap_servers", required = true) String bootstrapServers,
+public record TargetCluster(@JsonProperty(value = "bootstrapServers", required = true) @JsonAlias("bootstrap_servers") String bootstrapServers,
```

Comment:
We normally add a @deprecated annotation with a since=<ver> value when we deprecated something.  This helps us find things to remove when the time comes.   We can't do that here, but I think adding a comment with the same information would help."

Reply:
It would also be nice to log a warning when someone uses a deprecated property, which `@JsonAlias` doesn't allow. We could use a separate property, which would allow us to use @Deprecated and warn logging.